readers/cassandra/messages.go

```diff
@@ -85,21 +87,21 @@ func (cr cassandraRepository) ReadAll(chanID string, offset, limit uint64, query
 	}
 
 	if err := iter.Close(); err != nil {
```

Comment:
Please ignore this error and move `iter.Close()` to line 44 of this file. So it should be `defer iter.Close()`.